Lib/test/test_configparser.py

```diff
@@ -760,6 +760,15 @@ def test_read_returns_file_list(self):
         cf = self.newconfig()
         parsed_files = cf.read([], encoding="utf-8")
         self.assertEqual(parsed_files, [])
+        # check when we pass invalid file names:
+        cf = self.newconfig()
+        parsed_files = cf.read([
+            '\x00',                 # NUL bytes filename
+            __file__ + '\x00.ini',  # filename with embedded NUL bytes
+            "\uD834\uDD1E.ini",     # surrogate codes (MUSICAL SYMBOL G CLEF)
+            'a' * 1_000_000         # very long filename
```

Comment:
Do we know that 1M characters is always a ValueError? Is there a shorter length that's also always a ValueError? Windows has a max length of 32k (assuming long paths are enabled). Some resources indicate that Linux has a limit of 4k and macOS 1k. Maybe `2**16` would be better here.

Reply:
Since this value is a copy/paste from another test, let's just re-use what's already defined there.

Reply:
Err... I actually used the same number everytime in the other PRs, so I'd prefer not to change it (I'm always using the same dataset). I think I picked 1M because `1_000_000` was nice to write...